### ISSUE TITLE
Don't show two alerts on phase card when activating a phase

### ISSUE THREAD

**Issue by RishiRajSahu** [appirio-tech/connect-app#2489]
Hide the the Delete phase warning when marking the phase `In Progress` . 

![screen shot 2018-09-07 at 3 01 01 pm](https://user-images.githubusercontent.com/11248478/45288472-642f7e00-b508-11e8-995d-73d4ba647ae9.png)

**Comment by amitpatra** [comment]
I would like to work on it

**Comment by amitpatra** [comment]
Hi, I am working on it. Just wanna make sure that I am in the right direction. At this moment when we choose 'In Progress' as the status and click 'update phase' both the waring about activating the phase and an option to delete the phase are shown. I guess we need to hide the option to delete the phase right?

**Comment by maxceem** [comment]
@amitpatra Yes, you are right. In case we show warning about `activating the phase` the delete option has to be hidden. In all other cases where delete option is visible now it has to stay visible.

**Comment by amitpatra** [comment]
@maxceem made a pull request. Please review it. Thanks :)

**Comment by maxceem** [comment]
@RishiRajSahu In the provided PR the placement of the Delete Warning has been slightly changed

![image](https://user-images.githubusercontent.com/146016/45469040-5bce8180-b75a-11e8-991d-423e935e9e66.png)

So it become not the full width as it was before, see https://github.com/appirio-tech/connect-app/issues/2489#issue-358536164

IMHO it's even better now after this change, as all elements are uniformly aligned now. Also provided marvel doesn't match the current approach of displaying Phase editing form https://marvelapp.com/8a94680/screen/44224503

So I would like to approve the PR with this change. Let me know if it's not desirable.

**Comment by RishiRajSahu** [comment]
@maxceem delete phase warning looks good to me just take care of border radius as per the design. If the Activate phase alert is also coming below this dotted line only ? can we have a screenshot of that too ?

**Comment by maxceem** [comment]
@RishiRajSahu 

> If the Activate phase alert is also coming below this dotted line only ? can we have a screenshot of that too ?

It comes without the dotted line. Though as for me it feels ok, as there are no `Cancel`/`Update Phase` buttons too. So there is nothing to saparate from. And this Warning is shown insdtead of `Cancel`/`Update Phase` buttons rather then next to them.

![image](https://user-images.githubusercontent.com/146016/45470839-c9ca7700-b761-11e8-8b74-f2faa40721f3.png)

> just take care of border radius as per the design

Note, warning message also doesn't have border radius. If we want to add them I think I will create another issue for it to add for both places.

**Comment by RishiRajSahu** [comment]
okay this implementation is fine for now !

**Comment by maxceem** [comment]
@amitpatra Your PR is very good, just one small thing, please have a look https://github.com/appirio-tech/connect-app/pull/2497#pullrequestreview-154951142

**Comment by amitpatra** [comment]
Yes will fix that issue. Thanks :)
